### ISSUE TITLE
Add a platform or console icon related to the game

### ISSUE THREAD

**Issue by AAide** [JosefNemec/Playnite#2725]
### Feature description

It is possible to add in a future update an icon of a PSX on the PSX games, or a Steam icon on the steam games and other useful thing directly on the games in question. Because when in the list of all games, sometimes have the same games on several platforms and with the platform icon system in question, it allows to differentiate them and know what platform the game is.

**Comment by UnionExxis** [comment]
Yeah, this would be amazing. Really, really handy for emulation to know at a glance which platform you're using.

I imagine this feature working by setting an icon for the platform and then it would overlay it on all your cover art, similar to the platform images that appear in Fullscreen mode when coming from a storefront (like Steam or GOG).

There is another issue which I think may be the same feature request: https://github.com/JosefNemec/Playnite/issues/1450
